### DISCUSSION TITLE
Lgsl + XenForo

### DISCUSSION THREAD

**Discussion by Kruglik777** [tltneon/lgsl#124] [Q&A]
Good afternoon. How to display monitoring of specific servers on the Xenforo site?

**Comment by BaxAndrei** [comment]
This might help: https://github.com/tltneon/lgsl/wiki#fast-start-using-lgsl-as-php-library

**Comment by Kruglik777** [comment]
this is too hard for me)

**Comment by BaxAndrei** [comment]
![image](https://user-images.githubusercontent.com/12484505/183059303-28d15158-e93e-4484-89e5-aa2e707cd513.png)
None of the codes offered by the website are useful for you?

**Comment by Kruglik777** [comment]
I just wanted to display monitoring separately, like in IPB
https://ibb.co/1bRHfRF

**Comment by BaxAndrei** [comment]
You should use api from above and JS
